### PR TITLE
wdio-sauce-service: Use strictSSL option in requests to Sauce Labs

### PR DESCRIPTION
**Pull request by martinfrancois**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)
This one is kind of related to #3661, but applies it also to requests in the Sauce Labs service. 
Without this change, after running a test on Sauce Labs, the status of the job is not updated when behind a corporate proxy since the following error occurs in `updateVmJob` and `updateRdcJob`: `Error: unable to verify the first certificate`. However, since the error is simply rejected without handling or logging it, this was quite difficult to find. This is why I also added an error to be logged in these cases, should another connection-related error appear.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)
This may not be the best implementation, since it duplicates logic from #3661, which is also why I didn't add tests for the method, since it's already being tested there. It would probably be best to do a refactoring of this to a separate package, but I didn't know where the best place was to do this. Feel free to either tell me where to refactor it to or to merge it anyways and refactoring it later, since it may be a bit overkill at this stage, considering there is nothing else that is duplicated.

### CODE REVIEW

File: packages/wdio-sauce-service/src/service.js [hunk 1]
```diff
@@ -236,4 +240,12 @@ export default class SauceService {
         body.passed = failures === 0
         return body
     }
+
+    /**
+     * if the environment variable "STRICT_SSL" is defined as "false", it doesn't require SSL certificates to be valid.
+     */
+    getStrictSsl() {
```
Comment (christian-bromann) [inline packages/wdio-sauce-service/src/service.js]: can we make this a utility method?

Comment (martinfrancois) [reply packages/wdio-sauce-service/src/service.js]: Sure, will do!

**Comment by christian-bromann** [comment]
I know that you have already been working on this but I have to close it due to #3757. Would you mind making this PR in the [`node-saucelabs`](https://github.com/saucelabs/node-saucelabs) repository?

**Comment by martinfrancois** [comment]
@christian-bromann Just did so, see https://github.com/saucelabs/node-saucelabs/pull/54

**Pull request closed without merging**